client/src/components/Card.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+export default function Card () {
+  const [productInfo, setProductInfo] = useState([])
+
+  const getProductInfo = () => {
+    axios.get('/product/productid')
```

Comment:
Concern with adding too much "temporary" axios calls on client side. perhaps our priority should be to create the serverside axios calls first?

Reply:
@erikeh  Yea most of the calls will be on the server side primarly two types the styles and product. Biggest thing thou is For a single card it needs data from both styles and product then it needs then for Each card so gonna be quite a bit forEach kindof of calls with async and awaits mayber....

Reply:
Recommend creating a reusable method that gets data from both styles and product via promise.all() and returns combined data in useful format. 

That way every time you need to get both pieces of data you can just do something like

getCardData() 
  .then()....